### PR TITLE
Update dependencies (first batch)

### PR DESCRIPTION
**Pull request by LarsTheGlidingSquirrel**
## 1. Two major version bumps: 

**@next/bundle-analyzer**
Seems to work fine on pull request page. Any other place to check?  

**@svgr/webpack**
New config recommended by documentation of svgr. Tested and confirmed that svgs appear the same as before.

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -32,6 +44,7 @@ module.exports = withBundleAnalyzer({
   // TODO: reactStrictMode with react18 breaks edtr.io atm (inside react-hotkeys)
   reactStrictMode: false,
   productionBrowserSourceMaps: true,
+  transpilePackages: ['ramda'],
```
Comment (elbotho) [inline next.config.js]: why is this necessary?

**Review by elbotho — APPROVED** [review]
Thanks!

File: next.config.js (hunk 1 above)
Comment (Entkenntnis) [reply next.config.js]: for context: https://github.com/vercel/next.js/issues/40183

Comment (elbotho) [reply next.config.js]: thanks!
